docs/cloud-native-security/d4c-policy-guide.asciidoc

```diff
@@ -0,0 +1,94 @@
+|===
+
+[discrete]
+=== File-selector conditions
+These conditions are available only for `file` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `createExecutable`, `modifyExecutable`, `createFile`, `modifyFile`, `deleteFile`.
+| ignoreVolumeMounts | If set, ignores file operations on ALL volume mounts.
+| ignoreVolumeFiles | If set, ignores operations on file mounts only. For example: mounted files, `configMaps`, and secrets.
+| targetFilePath | A list of file paths to include. Paths are absolute and wildcards are supported. The `*` wildcard matches any sequence of characters within a single directory, while the `**` wildcard matches any sequence of characters across multiple directories and subdirectories.
+|===
+
+NOTE: In order to ensure precise targeting of file integrity monitoring operations, a `TargetFilePath` is required whenever the `deleteFile`, `modifyFile`, or `createFile` operations are used within a selector.
+
+[discrete]
+=== Process-selector conditions
+These conditions are available only for `process` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `fork` and `exec`.
+| processExecutable | A list of executables (full path included) to match on. For example: `/usr/bin/cat`. Wildcard support is same as targetFilePath above.
+| processName | A list of process names (executable basename) to match on. For example: `bash`, `vi`, `cat`.
+| sessionLeaderInteractive | If set to true, will only match on interactive sessions (i.e. sessions with a controlling TTY).
+|===
+
+[discrete]
+=== Response fields
+A policy can include one or more responses. Each response is comprised of the following fields:
+
+[cols="1,1", options="header"]
+|===
+| Field | Description
+| match | An array of one or more selectors of the same type (`file` or `process`).
+| exclude | Optional. An array of one or more selectors to use as exclusions to everything in `match`.
+| actions | An array of actions to perform when at least one `match` selector matches and none of the `exclude` selectors match. Options include `log`, `alert`, and `block`.
+|===
+
+[discrete]
+=== Response actions
+D4C responses can include the following actions:
+
+[cols="1,1", options="header"]
+|===
+| Action | Description
+| log | Sends events to the logs-cloud_defend.file-* data stream for file responses, and the logs-cloud_defend.process-* data stream for process responses.
+| alert | Writes events (file or process) to the logs-cloud_defend.alerts-* data stream.
+| block | Prevents the system operation from proceeding. This blocking action happens prior to the execution of the event. It is required that the alert action be set if block is enabled. Note: Currently block is only supported on file operations. Process blocking coming soon!
```

Comment:
Removed the last part since we don't normally promise future functionality in feature docs or release notes.
 
```suggestion
| block a| Prevents the system operation from proceeding. This blocking action happens prior to the execution of the event. It is required that the alert action be set if block is enabled. 

**Note:** Currently block is only supported on file operations. 
```